mobile_app_connector/models/res_partner.py

```diff
@@ -27,9 +27,24 @@ class GetPartnerMessage(models.Model):
         required=True,
     )
 
+    app_messages = fields.Many2one(
+        "mobile.app.messages", "Mobile app messages"
+    )
+
     ##########################################################################
     #                             PUBLIC METHODS                             #
     ##########################################################################
+    @api.model_create_multi
+    def create(self, vals_list):
+        partners = super().create(vals_list)
+
+        for partner_id in partners:
+            app_messages = self.env["mobile.app.messages"].create({
+                "partner_id": partner_id.id
+            })
+            partner_id.app_messages = app_messages
```

Comment:
Because it is minimizes a lot the risk to include a bug in the code but for most of the users (as they don't have the app), only their id will be stored, so the table will still keep a really small size. But yeah, a better solution would be to create a migration script to add one for all users that utilizes the app and to add an entry in the table at the creation of such a user. Otherwise, we could add an entry when trying to access it for a user if there is none.